go.mod

```diff
@@ -57,15 +57,14 @@ require (
 	github.com/uber/jaeger-lib v1.5.0 // indirect
 	github.com/unrolled/render v0.0.0-20180914162206-b9786414de4d // indirect
 	go.etcd.io/etcd v0.5.0-alpha.5.0.20191023171146-3cf2f69b5738
-	go.uber.org/atomic v1.4.0
-	go.uber.org/multierr v1.2.0 // indirect
-	go.uber.org/zap v1.10.0
+	go.uber.org/atomic v1.5.0
+	go.uber.org/zap v1.12.0
```

Comment:
Why update zap?

Reply:
It is updated automatically when I  updated `github.com/pingcap/check`